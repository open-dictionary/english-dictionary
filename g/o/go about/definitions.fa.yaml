- definition: گشتن
- definition: دور زدن
- definition: تغيير جهت دادن
- definition: (مج ).شايع بودن
- definition: تقلا کردن
- definition: مشغول شدن به
