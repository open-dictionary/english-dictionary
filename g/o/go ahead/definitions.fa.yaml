- definition: " :)adj.(متهور"
- definition: مترقى
- definition: پيش رونده
- definition: نشانه ترقى
- definition: بفرماييد
- definition: green light :)n.(
