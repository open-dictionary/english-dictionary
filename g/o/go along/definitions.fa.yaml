- definition: همراه رفتن
- definition: همراهى کردن
- definition: (مج ).موافق بودن
