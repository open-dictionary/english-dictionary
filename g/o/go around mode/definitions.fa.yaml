- definition: روش کنترل مخصوص کندن هواپيما از زمين روش کنترل صعود هواپيما
