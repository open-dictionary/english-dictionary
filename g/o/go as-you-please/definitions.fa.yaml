- definition: ول
- definition: ازاد
- definition: بى قيد
