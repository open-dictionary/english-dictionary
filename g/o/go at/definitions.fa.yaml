- definition: جدا مشغول شدن به
