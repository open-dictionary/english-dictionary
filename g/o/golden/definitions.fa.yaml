- definition: طلائى
- definition: طلايى
- definition: زرين
- definition: اعلا
- definition: درخشنده
